langchain/embeddings/openai.py

```diff
@@ -0,0 +1,54 @@
+"""Wrapper around OpenAI embedding models."""
+import os
+from typing import Any, Dict, List
+
+from pydantic import BaseModel, Extra, root_validator
+
+from langchain.embeddings.base import Embeddings
+
+
+class OpenAIEmbeddings(BaseModel, Embeddings):
+    """Wrapper around OpenAI embedding models."""
+
+    embedding_func: Any  #: :meta private:
+    model_name: str = "babbage"
+    """Model name to use."""
+
+    class Config:
+        """Configuration for this pydantic object."""
+
+        extra = Extra.forbid
+
+    @root_validator()
+    def validate_environment(cls, values: Dict) -> Dict:
+        """Validate that api key and python package exists in environment."""
+        if "OPENAI_API_KEY" not in os.environ:
+            raise ValueError(
+                "Did not find OpenAI API key, please add an environment variable"
+                " `OPENAI_API_KEY` which contains it."
+            )
+        try:
+            from openai.embeddings_utils import get_embedding
+
+            values["embedding_func"] = get_embedding
+        except ImportError:
+            raise ValueError(
+                "Could not import openai python package. "
+                "Please it install it with `pip install openai`."
```

Comment:
This error message was a bit misleading in the end cause I had openai installed & was missing the dependencies specifically for the embedding_utils import... wonder if changing the text here to be more specific about the import to test will help if others run into this problem?